packages/core/src/build/schema.ts

```diff
@@ -129,6 +159,8 @@ export const buildSchema = ({
         }
       }
 
+      // Note: Ponder lets postgres handle the table name length limit and truncation.
```

Comment:
What about when we append the `_reorg...` prefix - are we still vulnerable to conflicts after truncation?

I figured that we would need to manually enforce an identifier limit that's less than 63 characters, to leave room for prefixes/suffixes.

Reply:
When we append the `_reorg` prefix it's still possible to have a table name collision. If this happened there would be a non-retryable error emitted in the `migrate` function. 

We could enforce a limit that is less than 63 characters but I thought it might cause more harm than good when interfacing with external systems.

Reply:
IMO better to validate explicitly / fail fast. How short would we need to make it to leave room for all our possible prefixes and suffixes?